### DISCUSSION TITLE
Nginx manager on 2 servers

### DISCUSSION THREAD

**Discussion by MarcS1975** [NginxProxyManager/nginx-proxy-manager#2807] [Q&A]
Hi - I would like to have 2 nginx managers on 2 separate servers on my internal network, 1 live and 1 as manual failover. Manual meaning I will switch the router port redirection to Nginx#2, once I realise that nginx#1 is down.

My question:
Which data (certificates etc) do I need to share between the 2 nginx manager installations and where exactly is it stored?

**Comment by pujux** [comment]
you should be fine with sharing just the data volume you mount onto the proxy manager container

**Comment by MarcS1975** [comment]
OK thanks for the idea. I will try that!!
